packages/modules/workflow-engine-redis/src/loaders/redis.ts

```diff
@@ -59,12 +61,14 @@ export default async ({
   }
 
   container.register({
+    isWorkerMode: asValue(moduleDeclaration.worker_mode !== "server"),
```

Comment:
NIT: This is a bit confusing as the actual check we do is `isWorkerOrSharedMode`. I think we have it elsewhere like this as well so it's fine to leave it, but for a minute here I was gonna comment that we also need to check the "shared" mode 😄

Reply:
yes I agree, I ve just kept it that way for consistency and we can do a pass on all of them later in one shot 👍